client/src/components/organisms/CreateAgendaModal.tsx

```diff
@@ -199,9 +199,17 @@ export const CreateAgendaModal: React.FC = () => {
               style={{ textDecoration: "none" }}
             >
               <Button w={270} h={38} onClick={onSubmit} disabled={!validated}>
-                <Text variant="boldtitle3" color="blue600">
-                  투표 생성하기
-                </Text>
+                {
+                  validated ? ( 
+                    <Text variant="boldtitle3" color="blue600">
```

Comment:
```suggestion
                    <Text variant="boldtitle3" color={validated ? "blue600" : "gray어쩌고"}>
```
바뀌는 게 컬러 뿐인 것 같아서 요렇게 하면 코드 중복도 줄고 더 깔끔할 것 같아요!!!